compiler/rustc_index/src/bit_set.rs

```diff
@@ -654,15 +654,19 @@ impl<T: Idx> BitRelations<ChunkedBitSet<T>> for ChunkedBitSet<T> {
 
 impl<T: Idx> BitRelations<HybridBitSet<T>> for ChunkedBitSet<T> {
     fn union(&mut self, other: &HybridBitSet<T>) -> bool {
-        // FIXME: this is slow if `other` is dense, and could easily be
-        // improved, but it hasn't been a problem in practice so far.
+        // FIXME: This is slow if `other` is dense, but it hasn't been a problem
+        // in practice so far.
+        // If a a faster implementation of this operation is required, consider
```

Comment:
tiny typo here and below
```suggestion
        // If a faster implementation of this operation is required, consider
```

Reply:
Ah, that's stupid, thanks for catching it! Fixes in https://github.com/rust-lang/rust/pull/94956